frontend/public/components/graphs/graph-empty.tsx

```diff
@@ -16,6 +16,6 @@ export const GraphEmpty: React.FC<GraphEmptyProps> = ({height = 180, icon = Char
 
 type GraphEmptyProps = {
   icon?: React.SFC<any>;
-  height?: number;
+  minHeight?: number;
```

Comment:
Do we need to change the prop name or is this more of an implementation detail? We could keep the current prop name and just set the min height style with the value.

Reply:
I had the same thought.